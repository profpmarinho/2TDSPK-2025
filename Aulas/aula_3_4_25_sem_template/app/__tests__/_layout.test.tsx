import React from 'react';
import { render } from '@testing-library/react-native';
import { NavigationContainer } from '@react-navigation/native';
import RootLayout from '../_layout';

test('renders RootLayout correctly', () => {
  const { toJSON } = render(
    <NavigationContainer>
      <RootLayout />
    </NavigationContainer>
  );
  expect(toJSON()).toMatchSnapshot();
});