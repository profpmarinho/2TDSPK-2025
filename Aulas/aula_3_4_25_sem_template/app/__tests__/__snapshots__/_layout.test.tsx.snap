// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders RootLayout correctly 1`] = `
<View>
  <Text>
    Root Layout
  </Text>
</View>
`;
